### PR TITLE
Rce fix

### PR DESCRIPTION
**Pull request by Anon-Artist**
### 📊 Metadata *
Arbitrary code exec vulnerability 

### ⚙️ Description *

Chartify is a Python library that makes it easy for data scientists to create charts. This package was vulnerable to Arbitrary code execution via Insecure YAML deserialization due to the use of a known vulnerable function load() in yaml.

### 💻 Technical Description *

This package was vulnerable to Arbitrary code execution due to the use of a known vulnerable function load() in YAML. Changing that to safe_load or using SafeLoader will fix the issue.

### 🐛 Proof of Concept (PoC) *

Install the package and run the below code:
```#!/usr/bin/env python3
from chartify._core.colors import *

payload = '''
!!python/object/new:type
  args: ["z", !!python/tuple [], {"extend": !!python/name:exec }]
  listitems: "__import__('os').system('calc.exe')"
'''
open('pwn.yml', 'w+').write(payload)

CustomColors.from_yaml('hek', 'pwn.yml')
print("pwned 🏴")
```

### 🔥 Proof of Fix (PoF) *

After applying the fix, run the PoC again, `calc` wont pop and no code will be executed. Hence code exec is mitigated.

### 👍 User Acceptance Testing (UAT)

Only `SafeLoader` is used, which is the safer function, no breaking changes introduced.

### CODE REVIEW

**Review by mzfr — COMMENTED** [review]
LGTM!

P.S: I understand that previously  normal `open` was used but I would suggest to change it to `with open`.

File: chartify/_core/colors.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def from_yaml(self, filename):
         """Load colors from yaml file"""
         # Note: We assume that the contents of the config file are trusted
         # TODO: Change this file format to be plain yaml and use SafeLoader
```
Comment (Mik317) [inline chartify/_core/colors.py]: Remove `TODO` comment since fix is making this changes :+1:

**Review by Mik317 — APPROVED** [review]
LGTM :smile: :cake:

Cheers,
Mik

**Review by Anon-Artist — COMMENTED** [review]
Done @Mik317

**Comment by huntr-helper** [comment]
Congratulations Anon-Artist - your fix has been selected! 🎉

Thanks for being part of the community & helping secure the world's open source code.
If you have any questions, please respond in the comments section, or hit us up on Discord. Your bounty is on its way - keep hunting!

[Come join us on Discord](https://huntr.dev/discord)

**Pull request merged**